app/web/features/messages/requests/HostRequestView.tsx

```diff
@@ -186,7 +195,6 @@ export default function HostRequestView({
           </div>
         )}
       </UserSummary>
-      {isMobile && <Divider spacing={1} />}
```

Comment:
Decided to make these a border instead otherwise it had a little floating bit of padding after the divider.